### PR TITLE
Switch to noting datatests in PR comments

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
Instead of having a test that always fails, Github Actions means we can comment on the PR saying the user should run the command.

**Testing**
We should see it in this PR.

### CODE REVIEW

**Comment by stephenroller** [comment]
```
$ python tests/datatests/test_new_tasks.py
.
----------------------------------------------------------------------
Ran 1 test in 622.191s

OK
python tests/datatests/test_new_tasks.py  564.13s user 52.16s system 98% cpu 10:25.14 total
```

File: tests/datatests/test_new_tasks.py [hunk 1]
```diff
@@ -32,44 +31,45 @@ def test_verify_data(self):
...
+                    and issubclass(c, Teacher)
+                    and 'BadExampleTeacher' not in x
+                    and not x.startswith('_')
+                    and x
+                    not in [
+                        'Teacher',
+                        'MultiTaskTeacher',
+                        'FixedDialogTeacher',
+                        'DialogTeacher',
+                    ]
```
Comment (stephenroller) [inline tests/datatests/test_new_tasks.py]: this all feels wrong to me

Comment (stephenroller) [reply tests/datatests/test_new_tasks.py]: I need input from others here.

**Comment by stephenroller** [comment]
Just for you @EricMichaelSmith.

File: .circleci/config.yml [hunk 2]
```diff
@@ -139,29 +139,6 @@ buildwebsite: &buildwebsite
 # Actual jobs
 # -------------------------------------------------------------------------------------
 jobs:
-  datatests:
```
Comment (EricMichaelSmith) [inline .circleci/config.yml]: 👍

File: tests/datatests/test_new_tasks.py [hunk 3]
```diff
@@ -58,7 +60,8 @@ def test_verify_data(self):
                 opt = parser.parse_args(args=['--task', subt], print_args=False)
                 opt['task'] = subt
                 try:
-                    text, log = verify(opt, print_parser=False)
+                    with testing_utils.capture_output():
```
Comment (EricMichaelSmith) [inline tests/datatests/test_new_tasks.py]: This is great!

File: tests/datatests/test_new_tasks.py [hunk 4]
```diff
@@ -68,7 +71,11 @@ def test_verify_data(self):
                         print('There are {} {} in {}.'.format(log[key], key, subt))
                         found_errors = True
 
-        self.assertFalse(found_errors, "Errors were found.")
+        if found_errors:
+            self.fail(
```
Comment (EricMichaelSmith) [inline tests/datatests/test_new_tasks.py]: Yes, this is friendlier

**Pull request merged**